test/unit.js

```diff
@@ -968,6 +969,26 @@ describe('Populator', function () {
   });
 });
 
+describe('Populator with specified unique tracked entity attribute ID', function () {
+  describe('with multiple unique attributes', function () {
+    it('should set specified uniqueTrackedEntityAttributeID to the cache', async function () {
+      var populatorOptions = {
+        uniqueAttributeID: 'key3'
+      }
+      var populator = await new Populator(populatorOptions);
```

Comment:
The versions of Node that are currently supported do not support async/await so the build is failing. I'm not sure why this constructor call is being awaited in the first place either? Please remove the async/await.